### ISSUE TITLE
Support argo submit output to be in json format

### ISSUE THREAD

**Issue by shrinandj** [argoproj/argo#604]
Argo submit shows the result of the command in a nice tabular format. However, for running `argo submit` in a script, it is a bit hard to parse the output from the tabular format. Instead, it'll be good to have the output of `argo submit` in the json format.

**Comment by jessesuen** [comment]
One caveat. `argo submit` supports submission of multiple manifests in different files, or even in the same file separated by `---`. So the return value would be a list of workflow resources.
